src/index.html

```diff
@@ -13,13 +13,22 @@ <h1>LEADBOARD SCORE BOARD</h1>
       <input type="text" id="name" required />
       <label for="score">Score:</label>
       <input type="number" required id="score" />
-      <button type="submit">Submit</button>
+      <button type="submit" id="submitUser">Submit</button>
     </form>
     <div id="errorMessage"></div>
 
     <!-- score container -->
     <div id="score-board">
-      <p>Here I will implement the score board</p>
+      <button id="btnRefresh">Refresh</button>
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Points</th>
+          </tr>
+        </thead>
+        <tbody id="tableBody"></tbody>
+      </table>
```

Comment:
- [ ] Please let's try to follow the given wireframe below as it is a very big part of the requirements 👍 
### Wireframe
![image](https://github.com/Arslan49sp/leaderboard-micro/assets/46839548/8c290abc-0b59-4aba-a39e-382066864a02)
### Your Design
![image](https://github.com/Arslan49sp/leaderboard-micro/assets/46839548/d89f548d-a072-46c6-a165-eb0f1ac9f653)

Reply:
I kindly request to approve this projct because the  task to design the webpage is in my next project in alignment with the Microverse instructions. My primary objective in this project is to connect to the API and ensure the smooth functioning of score storage and retrieval features.

Reply:
@codepantha These were the project requirements 

- Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
- Create a new game with the name of your choice by using the API.
- Make sure that you saved the ID of your game that will be returned by API.
- Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the API.
- No styling is required.

As you can see at the last that no styling was required. So will do this styling in the next project. In this project I just added markup. I'll structure that using CSS.